apollo-compiler/src/main/kotlin/com/apollostack/compiler/QueryTypeSpecBuilder.kt

```diff
@@ -0,0 +1,90 @@
+package com.apollostack.compiler
+
+import com.apollostack.api.GraphQLQuery
+import com.apollostack.compiler.ir.Fragment
+import com.apollostack.compiler.ir.Operation
+import com.squareup.javapoet.*
+import java.util.*
+import javax.lang.model.element.Modifier
+
+class QueryTypeSpecBuilder(
+    val operation: Operation,
+    val fragments: List<Fragment>
+) {
+  fun build(): TypeSpec {
+    val queryClassName = operation.operationName.capitalize()
+    val queryInterfaceTypeName = ParameterizedTypeName.get(ClassName.get(GraphQLQuery::class.java),
```

Comment:
sounds like we should create a constant for `ClassName.get(GraphQLQuery::class.java)`. If we have lots of queries, this will be done N times and we can avoid that, just like we're doing for the nullable/nonnull annotations